docs/server-versions-dependencies.md

```diff
@@ -33,17 +35,6 @@ This versioning policy is intended to serve as a minimum set of guidelines for c
 
 Temporal offers official support for, and is tested against, dependencies with the exact versions described below.
 
-### Go Packages
-
-| Temporal version | Go packages |
-|------------------|-------------|
-| [1.5.1](https://github.com/temporalio/temporal/tree/v1.5.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.5.1/go.mod) |
-| [1.4.2](https://github.com/temporalio/temporal/tree/v1.4.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.4.2/go.mod) |
-| [1.3.2](https://github.com/temporalio/temporal/tree/v1.3.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.2/go.mod) |
-| [1.2.2](https://github.com/temporalio/temporal/tree/v1.2.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.2.2/go.mod) |
-| [1.1.1](https://github.com/temporalio/temporal/tree/v1.1.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.1.1/go.mod) |
-| [1.0.0](https://github.com/temporalio/temporal/tree/v1.0.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.0.0/go.mod) |
-
```

Comment:
Per Alex & Wenquan - we could update this to just say something like:

"Temporal offers official support for, and is tested against, dependencies with the exact versions described in the `go.mod` file of corresponding release tag."

Reply:
done